index.html

```diff
@@ -40,11 +40,85 @@
+
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <img src="./IMG_1942.jpeg" alt="Placeholder Image" />
+      
+      <h1>Papillion - the Butterfly dogs </h1>
+
+      <p>
+        The Papillion is a breed of dog derived from the European toy spaniel, being named in the 1930's. 
+        There name is derived from the French word for butterfly, due to the upright shape of their ears, looking like the shape of butterfly wings. There is also a more common variant, called the phalene, named after the french name for a type of moth. due to the droopy nature of there ears more akin to a moth than that of a butterfly. 
+      </p>
+      <p>
+        They are known as a common breed of lapdogs, in the us being ranked as the 51st most common dog breed in 2022, due to there reasonably calm annd friendly nature as a breed, though of course there are exceptions. As they are also a reasonably high energy breed, but adaptable to being either more outdoorsy or more indoors of an animal</p>
+      <br>
+      <h3>Details of the breed</h3>
+      <p>
+        The breed is generally a long living breed living to around 15 years old, or well into there 70's. 
+        As well as due there energetic nature they generally remain energetic through out there life seemingly drawing from a fountain of youth, and slow down little in old age.
+      </p>
+      <p>
+         There hair is reasonably long and of a more silky type of hair, and as a result do require regular grooming to keep there hair in good condition, and to prevent matting, to keep it in good shape and looking its best.  
+      </p>
+
+      <p>
+        More gennerally however they are a mostly healthy breed of dogs, enjoying the health and life span benefits that smaller dogs generally do over larger breeds of dogs, as well are free of the health issues that less common breeds of dogs can have. The mkain issues that they do have relate to poor knees and issues with blindness, especially through degredation of there retinas.</p>
+
+        <img src="./IMG_5136.jpeg" alt="Placeholder Image" />
+
+
+      <h3>Personality</h3>
+
+      <p>
+        There reputation as a lapdog comes from there calm and friendly nature, getting along well with children and other animals, including cats. They are fairly intelligent a breed as well and thus are easy to train which helps with there reputation as a good family dog, and one thats good in larger homes or city apartments. They also are a generally social breed and want to be around there owners commonly, and are eager to spend time near and around them.
+      </p>
+
+
+      <form>
+        <label for="name">Name</label>
+        <input type="text" name="name" id="name" />
+
+        <label for="email">Email</label>
+        <input type="email" name="email" id="email" />
+
+        <label for="message">Tell us about your experiences</label>
+        <textarea name="message" id="message"></textarea>
+
```

Comment:
Would have liked to have seen some autocomplete and other form attributes to make your form a bit better